src/index.js

```diff
@@ -1 +1 @@
-console.log('application works ');
\ No newline at end of file
+import './style.css';
```

Comment:
Your app does not contain any of the functionality required by the project. Kindly add the following functionalities:
- [ ] Kindly Create an index.js file and set an array of some simple to-do tasks (array of objects). Each task object should contain three keys:
- description [string].
- completed [bool].
- index: [number].

- [ ] Write a function to iterate over the tasks array and populate an HTML list item element for each task.

- [ ] On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.

Reply:
@topeogunleye okay let me do that now sorry i forgot

Reply:
@topeogunleye i have made the required changes

Reply:
@Bria222, Kindly submit for a rereview after making the requested changes.